src/Kiota.Builder/Extensions/OpenApiReferenceExtensions.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.OpenApi.Models;
+
+namespace Kiota.Builder {
```

Comment:
given it's under the extensions folder shouldn't the namespace be Kiota.Builder.Extensions?

Reply:
fair, I'll rename that in my latest PR to avoid having to handle conflicts all over.